x-pack/solutions/security/plugins/security_solution/public/configurations/links.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ConfigurationTabs } from './constants';
+
+import type { LinkItem } from '..';
+import { CONFIGURATIONS_PATH, SECURITY_FEATURE_ID, SecurityPageName } from '../../common/constants';
+import { CONFIGURATIONS } from '../app/translations';
+
+export const configurationsLinks: LinkItem = {
+  capabilities: [[`${SECURITY_FEATURE_ID}.show`, `${SECURITY_FEATURE_ID}.configurations`]],
+  globalNavPosition: 3,
+  globalSearchKeywords: [
+    i18n.translate('xpack.securitySolution.appLinks.configurations', {
+      defaultMessage: 'Configurations',
+    }),
+  ],
+  hideTimeline: true,
+  id: SecurityPageName.configurations,
+  path: CONFIGURATIONS_PATH,
+  title: CONFIGURATIONS,
+  links: [
+    {
+      id: SecurityPageName.configurationsIntegrations,
+      title: i18n.translate('xpack.securitySolution.appLinks.integrations', {
```

Comment:
NIT: For the 3 nested link titles, we could reuse the translations at  
`x-pack/solutions/security/plugins/security_solution/public/configurations/translations.ts`, so we do not duplicate.